symphony-bdk-core/src/main/java/com/symphony/bdk/core/ServiceFactory.java

```diff
@@ -4,7 +4,7 @@
 import com.symphony.bdk.core.client.ApiClientFactory;
 import com.symphony.bdk.core.config.model.BdkConfig;
 import com.symphony.bdk.core.retry.RetryWithRecoveryBuilder;
-import com.symphony.bdk.core.service.MessageService;
+import com.symphony.bdk.core.service.message.MessageService;
```

Comment:
This breaking change might impact our documentation even if I don't think that we specify packages, please take a look just in case. 

But it will for sure break `generator-symphony`: 
- https://github.com/SymphonyPlatformSolutions/generator-symphony/blob/master/generators/bdk/java/templates/spring/GifSlashHandler.java.ejs#L6
- https://github.com/SymphonyPlatformSolutions/generator-symphony/blob/master/generators/bdk/java/templates/spring/OnUserJoinedRoomListener.java.ejs#L3
- https://github.com/SymphonyPlatformSolutions/generator-symphony/blob/master/generators/bdk/java/templates/common/GifFormActivity.java.ejs#L8

Please create a ticket to update this repo accordingly.